src/main/java/com/gmail/axelwerst/Shape.java

```diff
@@ -0,0 +1,76 @@
+package com.gmail.axelwerst;
```

Comment:
Formatting is off. Please auto-format all of your files before submitting the PR for review.